js/characters.js

```diff
@@ -0,0 +1,17 @@
+const $apiContainer = document.querySelector('.api-container');
+
+const request = new XMLHttpRequest();
+request.open('GET', 'https://api.potterdb.com/v1/characters/harry-potter');
+request.responseType = 'json';
+request.send();
+
+function loadEvent() {
+  // console.log('response of http request:', request.response);
+
+  for (let i = 0; i < request.response.data.length; i++) {
+    const $li = document.createElement('li');
+    $li.textContent = request.response[i].name;
+    $apiContainer.appendChild($li);
+  }
+}
+request.addEventListener('load', loadEvent);
```

Comment:
I don't think this is even being used? You're making the request in `main.js` for all the characters, so what's going on with this?